run.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 
-docker run -dit -p 3000:3000 -e HTTPS=0 --restart always orb-server
+docker run -dit -p 3000:3000 -e HTTPS=0 --restart always --env "DB_HOST" --env "DB_PORT" --env "DB_USER" --env "DB_PWD" --env "DB_NAME" --env-file src/app/config/orb-server.env -v $(pwd)/src:/orb-server/src orb-server
```

Comment:
No need to add `--env` if we are providing an `env-file`, no?

Reply:
I took out the --env before each variable and there is a docker error.

Reply:
Only the `--env` preceding each variable?  There will certainly be an error if you give all of these variable names without any indication of what they mean (do you understand what each word in the command means?  Worth going over just to make sure you understand what you are writing).  I am arguing you can take out all of the following text:
`--env "DB_HOST" --env "DB_PORT" --env "DB_USER" --env "DB_PWD" --env "DB_NAME"`

Reply:
got it.